src/Illuminate/Http/Client/Factory.php

```diff
@@ -151,12 +151,13 @@ public function globalOptions($options)
      * @param  array|string|null  $body
      * @param  int  $status
      * @param  array  $headers
+     * @param  int  $flags
```

Comment:
You can narrow this further down to:
```suggestion
     * @param  int-mask-of<JSON_*>  $flags
```